<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWindow</class>
 <widget class="QMainWindow" name="CalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="calButton">
         <property name="text">
          <string>CALIBRATION</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="finishedButton">
         <property name="text">
          <string>Finished</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="homeButton">
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>textEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
